java/RSPEC-4524.java

```diff
@@ -0,0 +1,15 @@
+class SwitchDefaultNoFinal {
+    void test() {
+        switch (myVariable) {
+            default:
```

Comment:
RSPEC-4524: 'default' should be the line switch case
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._